firebase-firestore/src/main/java/com/google/firebase/firestore/model/value/ProtoValues.java

```diff
@@ -157,8 +164,8 @@ private static boolean objectEquals(Value left, Value right) {
   }
 
   /** Returns true if the Value list contains the specified element. */
-  public static boolean contains(List<Value> haystack, Value needle) {
-    for (Value haystackEl : haystack) {
+  public static boolean contains(ArrayValueOrBuilder haystack, Value needle) {
+    for (Value haystackEl : haystack.getValuesList()) {
```

Comment:
Nit: Java doesn't tend to abbreviate like this. Consider naming this just `element`, or `haystackElement` if you want to preserve the binding to the haystack.

Reply:
Used `haystackElement`